CONTRIBUTING

```diff
@@ -0,0 +1,11 @@
+How to contribute to MPICH
+
+1. Sign and submit a contributors' agreement. http://www.mpich.org/documentation/contributor-docs/
+
+2. Format your patches according to the MPICH coding standards. https://wiki.mpich.org/mpich/index.php/Coding_Standards
+
+3. Install this pre-commit hook (https://gist.github.com/wesbland/3546c4483ceffe129a6e7fc4105fbf75) in your local git repository to catch whitespace violations in new code. Patches with improper whitespace may be automatically rejected for inclusion in MPICH.
```

Comment:
Can we put a copy of it on the `mpich` github (with credit to Wesley's github)?

Reply:
Yeah, I was kinda thinking the same thing. While I love the glory that comes with creating the hook 🙄, I'd love to hand off the maintenance.

Reply:
Just coming back to this. Will do.

Reply:
Oh wait, we put the hook into the git repo. I should update the instructions.